freqtrade/exchange/exchange.py

```diff
@@ -585,6 +590,9 @@ def get_history(self, pair: str, ticker_interval: str,
     def refresh_latest_ohlcv(self, pair_list: List[Tuple[str, str]]) -> List[Tuple[str, List]]:
         """
         Refresh in-memory ohlcv asyncronously and set `_klines` with the result
+        Loops asyncroneously over pair_list and dowloads all pairs async (semi-parallel).
+        :param pair_list: List of 2 element tuples containing pair,interval to refresh
```

Comment:
space after the comma (yes, I'm a kind of grammar-nazi, sorry)

Reply:
that's fine - i tend to rush docstrings ... (i want spaces after commas, too)

Reply:
E231 missing whitespace after ','

😆